### ISSUE TITLE
always display the mutli login prompt for mutli accounts login

### ISSUE THREAD

**Issue by vipconsult** [thephpleague/oauth2-google#11]
If the user is currently logged in with more than one account it should always prompt which account to use.

Currently is always uses the first one in the list.

**Comment by shadowhand** [comment]
I don't think that is the case. I'm pretty sure that only happens if you have already authenticated with specific OAuth credentials and re-authenticate. To force a fresh authentication, you can pass the `approval_prompt=force` option to force displaying the approval screen.

**Comment by shadowhand** [comment]
To clarify, that means using:

```php
$url = $provider->getAuthorizationUrl([
    'approval_prompt' => 'force',
]);
```

**Comment by vipconsult** [comment]
this is for the authentication. 

I am talking after an app has been authorised 
 
I have created a pull request to fix the issue
https://github.com/thephpleague/oauth2-google/pull/13

**Comment by shadowhand** [comment]
The `approval_prompt` setting does what you want.

**Comment by shadowhand** [comment]
Fixed by #13
